src/clj/game/cards/events.clj

```diff
@@ -460,6 +460,29 @@
    "Easy Mark"
    {:msg "gain 3 [Credits]" :effect (effect (gain :credit 3))}
 
+   "Embezzle"
+   (letfn [(name-string [cards] (clojure.string/join " and " (map :title cards)))] ; either 'card' or 'card1 and card2'
```

Comment:
Do we not already have a util function like this? Feels like it would be used with cards trashed to damage and other multi-card reveal effects.

Reply:
Also, do we not import string/join, so you could use it just as `join`?